experiment_buddy/experiment_buddy.py

```diff
@@ -207,7 +207,7 @@ def deploy(host: str = "", sweep_yaml: str = "", proc_num: int = 1, wandb_kwargs
             extra_slurm_headers += "\n#SBATCH --partition=main"
 
         _commit_and_sendjob(host, experiment_id, sweep_yaml, git_repo, project_name, proc_num, extra_slurm_headers, wandb_kwargs)
-        sys.exit()
```

Comment:
the program should stop executing locally if the deployment has an host.
without the sys.exit the execution would not "continue" remotely but would continue in both machines

Reply:
I'd prefer to split the `deploy` in a `client_deploy` and a `server_deploy`, these are two different logic and should be treated as two different logic, how to be treated differently will be the journey for another PR

Btw, try again merging the `feature/time_logger`, true that it has an extra library, but I tested with `sys.exit()` and the log reaches the server no matter what (and I kept the `telemetry` api)

Reply:
client_deploy and a server_deploy, yes
as long as the user doesn't see this split

Reply:
The user will have one and only one API, buddy will take care of the rest, but I'd like to avoid like hell any kind of state machine embedded in the system